packages/flutter_tools/lib/src/ios/ios_deploy.dart

```diff
@@ -306,8 +304,8 @@ class IOSDeployDebugger {
         _launchCommand,
         environment: _iosDeployEnv,
       );
-      String lastLineFromDebugger;
-      final StreamSubscription<String> stdoutSubscription = _iosDeployProcess.stdout
+      String? lastLineFromDebugger;
+      final StreamSubscription<String>? stdoutSubscription = _iosDeployProcess?.stdout
```

Comment:
is `_iosDeployProcess` being null at this point a valid state?

Reply:
from looking at the code, I think this can be `_iosDeployProcess!.stdout`

Reply:
and i think `stdoutSubscription` can then be non-nullable

Reply:
Done.